solver/src/solver/http_solver.rs

```diff
@@ -293,10 +293,12 @@ impl HttpSolver {
         let now = chrono::Utc::now();
         format!(
             "{}_{}_{}",
-            now.to_string().replace(" ", "_"),
+            now.to_string(),
             self.network_name,
             self.chain_id
         )
+        .replace(" ", "_")
+        .replace("/", "_")
```

Comment:
Note that `replace` takes a `Pattern`, so you can:
```suggestion
        .replace(&[' ', '/'], "_")
```

Reply:
Somehow that doesn't work:

Reply:
I couldn't find a way to use replace with multiple chars but used a match statement instead. I guess it does only one pass as well.

Reply:
Sorry, this wasn't something that blocked approval on my part, just got distracted by something else.